src/transformers/models/electra/configuration_electra.py

```diff
@@ -104,6 +106,8 @@ class ElectraConfig(PretrainedConfig):
             <https://arxiv.org/abs/1803.02155>`__. For more information on :obj:`"relative_key_query"`, please refer to
             `Method 4` in `Improve Transformer Models with Better Relative Position Embeddings (Huang et al.)
             <https://arxiv.org/abs/2009.13658>`__.
+        classifier_dropout:
+            The dropout ratio for classifier.
```

Comment:
This needs a bit more, see the other arguments:
```suggestion
        classifier_dropout (:obj:`float`, `optional`):
            The dropout ratio for the classification head.
```